src/main/java/org/folio/dcb/service/impl/TransactionsServiceImpl.java

```diff
@@ -63,13 +63,10 @@ public TransactionStatusResponse getTransactionStatusById(String dcbTransactionI
 
   private TransactionStatusResponse generateTransactionStatusResponseFromTransactionEntity(TransactionEntity transactionEntity) {
     TransactionStatus.StatusEnum transactionStatus = transactionEntity.getStatus();
-    Role.TransactionRoleEnum transactionRole = transactionEntity.getRole();
```

Comment:
In get API, I think we need to return both status and role. So we need to keep this line.

Reply:
fixed